npm script "rollup" fails if no dist folder exists

**Describe the bug**
Correct if I am wrong, but the dist folder should contain the code built by the "rollup" command.
So, it should be possible to delete the dist folder and run the "rollup" command to do a clean build.
In this case, the command fails because do not find the dist/extensions folder when copying the *.php files.

**To Reproduce**
Steps to reproduce the behavior:
1. Open a terminal
2. cd to the folder where svg-edit code is
3. Delete the dist folder and all of its content
4. Execute "npm run rollup"

**Expected behavior**
I expert the command to run successfully and rebuild all the files in the dist folder, creating it if does not exist

**SVG-Edit environment (IMPORTANT)**
- Version: 5.0.0, but I tried also in the master branch
- Protocol: [e.g., http, https, file]

**Desktop (please complete the following information):**
 - OS: Windows 10

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Log**
> svgedit@6.0.0 rollup C:\svgedit-master
> rollup -c

[!] Error: ENOENT: no such file or directory, copyfile 'editor\extensions\allowedMimeTypes.php' -> 'dist\extensions\allowedMimeTypes.php'
Error: ENOENT: no such file or directory, copyfile 'editor\extensions\allowedMimeTypes.php' -> 'dist\extensions\allowedMimeTypes.php'
    at Object.copyFileSync (fs.js:1743:3)
    at extensionFiles.map (C:\svgedit-master\rollup.config.js:188:10)
    at Array.map (<anonymous>)
    at Object.<anonymous> (C:\svgedit-master\rollup.config.js:186:21)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.require.extensions.(anonymous function) [as .js] (C:\svgedit-master\node_modules\rollup\dist\bin\rollup:1809:24)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Module.require (internal/modules/cjs/loader.js:723:19)
    at require (internal/modules/cjs/helpers.js:14:16)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! svgedit@6.0.0 rollup: `rollup -c`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the svgedit@6.0.0 rollup script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

**Additional context**
Add any other context about the problem here.

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -4,7 +4,7 @@
 // See rollup-config.config.js instead for building the main (configurable)
 //   user entrance file
 import {join, basename} from 'path';
-import {lstatSync, readdirSync, copyFileSync} from 'fs';
+import {lstatSync, readdirSync, copyFileSync, mkdirSync} from 'fs';
 
 import babel from 'rollup-plugin-babel';
 import {terser} from 'rollup-plugin-terser';
@@ -183,6 +183,9 @@ export default [
   }),
   ...extensionFiles.map((extensionFile) => {
     if (extensionFile.match(/\.php$/)) {
+      mkdirSync('dist/extensions', { recursive: true }, (err) => {
+        if (err) throw err;
+      });
       copyFileSync(
         join('editor/extensions', extensionFile),
         join('dist/extensions', extensionFile)

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -183,9 +183,7 @@ export default [
   }),
   ...extensionFiles.map((extensionFile) => {
     if (extensionFile.match(/\.php$/)) {
-      mkdirSync('dist/extensions', { recursive: true }, (err) => {
-        if (err) throw err;
-      });
+      mkdirSync('dist/extensions', {recursive: true});
       copyFileSync(
         join('editor/extensions', extensionFile),
         join('dist/extensions', extensionFile)

